src/components/containers/Fold.js

```diff
@@ -49,15 +49,14 @@ export default class Fold extends Component {
           <CloseIcon />
         </div>
       );
-    const PlotIcon = plotIcons && plotIcons[type];
-    const plotTypeIcon = type &&
-      plotIcons && <PlotIcon className="fold__top__plot-icon" />;
+
+    const icon = Icon && <Icon className="fold__top__icon" />;
 
     const foldHeader = !hideHeader && (
       <div className={headerClass} onClick={this.toggleFold}>
         <div className="fold__top__arrow-title">
           {arrowIcon}
-          {plotTypeIcon}
+          {icon}
           <div className="fold__top__title">{name}</div>
```

Comment:
you know, I've found that when we don't explicitly return a null, in the event that `Icon && <Icon className="fold__top__icon" />;` is undefined, the `{icon}` in foldHeader will be rendered by react as a `0`.

At least that's what happens in our accordion menus now, since we're not returning null for these: https://github.com/plotly/react-plotly.js-editor/blob/master/src/components/containers/TraceAccordion.js#L69

They turn into a `zero` when we delete all traces. 

so could you have `icon ? icon : null` or some other variant up there, just to make sure.

for TraceAccordion, I fixed these in my collapse pr.

Reply:
I'll go through and remove those types of conditionals and replace with an explicit `null`